src/main/java/uk/gov/hmcts/dts/fact/config/security/SecurityConfiguration.java

```diff
@@ -18,7 +18,7 @@ protected void configure(HttpSecurity http) throws Exception {
                 .antMatchers(HttpMethod.GET, "/admin/**").authenticated()
                 .antMatchers(HttpMethod.GET, "/courts/").authenticated()
                 .antMatchers(HttpMethod.GET, "/courts/all").authenticated()
-                .antMatchers(HttpMethod.GET, "/courts/{slug}/*").authenticated()
+                .antMatchers(HttpMethod.GET, "/courts/{slug}/courtPhoto").authenticated()
```

Comment:
Might be a mistake?

Reply:
I have checked the /courts/{slug}/courtPhoto endpoint is the one that requires authentication as is still authenticated